### PR TITLE
Update Venetian.xml to 8.4.7

### LINKED ISSUE #12270: Venetian mistranslation: "Alternate icones"
### Description of the Issue

Venetian mistranslation
Settings-->Preferences-->general-->"Alternate icones" is mistranslated "Alterna icone"; the correct translation is "**Icone alternative**"

### Debug Information
Notepad++ v8.4.6   (64-bit)
Build time : Sep 25 2022 - 19:51:39
Path : C:\Program Files\Notepad++\notepad++.exe
Command Line : "C:\Program Files\Notepad++\change.log" 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
OS Name : Windows 11 (64-bit) 
OS Version : 22H2
OS Build : 22621.521
Current ANSI codepage : 1252
Plugins : 
    ComparePlugin (2.0.2)
    DSpellCheck (1.4.24)
    mimeTools (2.8)
    NppConverter (4.4)
    NppExport (0.4)
    NPPJSONViewer (1.40)
    NppXmlTreeviewPlugin (2)
    XMLTools (3.1.1.13)

immagine

### PR DESCRIPTION
**Pull request by conky77**
fix #12270

### CODE REVIEW

File: PowerEditor/installer/nativeLang/venetian.xml [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
 <!--
     Venetian translation for Notepad++
-    Last modified Mon, January 10th, 2022.
+    Last modified Wed, October 27th, 2022.
     Translators: 2019-xxxx, Matteo Concato (Conky77)
     For updates, see https://github.com/notepad-plus-plus/notepad-plus-plus/tree/master/PowerEditor/installer/nativeLang
     The comments are here for explanation, it's not necessary to translate them.
 -->
+
+
```
Comment (donho) [inline PowerEditor/installer/nativeLang/venetian.xml:10]: remove the added blank lines please.

File: PowerEditor/installer/nativeLang/venetian.xml [hunk 2]
```diff
@@ -395,13 +416,25 @@
...
+                <Item CMID="32" name="Còpia ne i apunti"/>
+                <Item CMID="33" name="Sposta scheda"/>
+                <Item CMID="34" name="Aplica Coƚori a ƚa scheda"/>
             </TabBar>
         </Menu>
+
         <Dialog>
             <Find title="" titleFind="Cata" titleReplace="Sostituisi" titleFindInFiles="Ruma inte i file" titleFindInProjects="Ruma inte&apos;l progeto" titleMark="Evidensia">
-                <Item id="1" name="Cata el Seguente"/>
+                <Item id="1"    name="Cata el Seguente"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/venetian.xml:435]: restore as it is.

File: PowerEditor/installer/nativeLang/venetian.xml [hunk 3]
```diff
@@ -395,13 +416,25 @@
...
                 <Item id="1722" name="Cata el Presedente"/>
-                <Item id="2" name="Sara su"/>
+                <Item id="2"    name="Sara su"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/venetian.xml:437]: Restore it

File: PowerEditor/installer/nativeLang/venetian.xml [hunk 4]
```diff
@@ -1162,8 +1253,8 @@ Te poi definire vari deƚimitatori de coƚona uxando spasi par spartire i divers
                 <Item id="2" name="Anuƚa"/>
             </ColumnEditor>
             <FindInFinder title="Cata tra i rixultadi">
-                <Item id="1" name="Cata tuti cuanti"/>
-                <Item id="2" name="Sara su"/>
+                <Item id="1"    name="Cata tuti cuanti"/>
+                <Item id="2"    name="Sara su"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/venetian.xml:1257]: restore as it is

File: PowerEditor/installer/nativeLang/venetian.xml [hunk 5]
```diff
@@ -1232,11 +1319,9 @@ Continuare?"/>
 Salvaƚo o i to dati i vegnerà persi.
 Vuto salvare el file &quot;$STR_REPLACE$&quot; ?"/>
             <DoReloadOrNot title="Recarga" message="&quot;$STR_REPLACE$&quot;
-
 Sto file el xe xà stà modifegà da &apos;n altro programa.
 Vuto recargarƚo?"/>
             <DoReloadOrNotAndLooseChange title="Recarga" message="&quot;$STR_REPLACE$&quot;
-
```
Comment (donho) [inline PowerEditor/installer/nativeLang/venetian.xml:1239]: Restore these deleted lines.

File: PowerEditor/installer/nativeLang/venetian.xml [hunk 6]
```diff
@@ -1246,11 +1331,13 @@ Vuto &apos;ndare inte ƚa pàgina de Notepad++ par descargare ƚ&apos;axornament
...
             <FileTooBigToOpen title="Problema dimension file" message="El file el xe masa grando par èsare vèrto co Notepad++"/> <!-- HowToReproduce: Try to open a 4GB file (it's not easy to reproduce, it depends on your system). -->
+            <FileLoadingException title="Codexe de ecesion: $STR_REPLACE$" message="Si è verificato un errore durante il caricamento del file!"/>
+            <WantToOpenHugeFile title="Vertaura file de grandi dimensioni" message="Par ƚa vertaura de file superiori a 2GB+ poderìa volerghe un mucio de minudi.
+Vuto ndar vanti?"/>
             <CreateNewFileOrNot title="Crea novo file" message="&quot;$STR_REPLACE$&quot; no existe. Crearƚo??"/>
             <CreateNewFileError title="Crea novo file" message="Inposibiƚe creare el file &quot;$STR_REPLACE$&quot;."/> <!-- HowToReproduce: this message prevents from system failure. It's hard to reproduce. -->
             <OpenFileError title="ERORE" message="Inposibiƚe vèrxare el file &quot;$STR_REPLACE$&quot;."/>
             <FileBackupFailed title="Backup File Faƚìo" message="Ƚa version presedente de&apos;l file no ƚa poƚe èsare salvà inte ƚa carteƚa de backup &quot;$STR_REPLACE$&quot;.
-
```
Comment (donho) [inline PowerEditor/installer/nativeLang/venetian.xml:1253]: restore it

**Review by donho — COMMENTED** [review]
No formatting
Please make only necessary modification.

**Comment by conky77** [comment]
If you have to close the version, please close it without Venetian,  I'll check and fix it in the next days, I can't today

**Comment by conky77** [comment]
I close this pull request, and I'll open another one after a review

**Pull request closed without merging**